pkg/reconciler/v1alpha1/route/resources/cluster_ingress_test.go

```diff
@@ -36,6 +36,9 @@ func TestMakeClusterIngress_CorrectMetadata(t *testing.T) {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "test-route",
 			Namespace: "test-ns",
+			Annotations: map[string]string{
+				"test-key": "test-value",
```

Comment:
Do we have the cluster ingress class annotation constant yet?  If so, it is worth specifically testing that it passes through.

Reply:
We do in https://github.com/knative/serving/pull/2477 - how about I add the test case there to avoid making the PR's inter-dependent.

Reply:
not sure what I was thinking - were going to be interdependent either way. I'll add it in here and we can wait until 2477 merges